client/protocols/backup.py

```diff
@@ -26,13 +27,26 @@ def run(self, messages):
             return
 
         message_list = self.load_unsent_messages()
-        message_list.append(messages)
 
         access_token = auth.authenticate(False)
         log.info('Authenticated with access token %s', access_token)
+        log.info('Sending unsent messages %s', access_token)
 
-        response = self.send_all_messages(access_token, message_list)
-        prefix='http' if self.args.insecure else 'https'
+        # Messages from the current backup to send first
+        subm_messages = [messages] if self.args.submit else []
+
+        if self.args.submit:
+            response = self.send_all_messages(access_token, subm_messages,
```

Comment:
Ah, I just got to `send_messages`. So it basically converts all stale submit runs into regular backups?